src/components/Microsoft.DotNet.UpgradeAssistant.Portability/Service/ApiInformation.cs

```diff
@@ -27,12 +27,12 @@ public bool IsSupported()
         {
             foreach (var supported in Supported)
             {
-                if (supported.Identifier.Contains("Standard", StringComparison.OrdinalIgnoreCase))
+                if (supported.Identifier.ToUpperInvariant().Contains("STANDARD"))
                 {
                     return true;
                 }
 
-                if (supported.Identifier.Contains("Core", StringComparison.OrdinalIgnoreCase))
+                if (supported.Identifier.ToUpperInvariant().Contains("CORE"))
```

Comment:
How is this better than StringComparison.OrdinalIgnoreCase?

Reply:
It's not, but it works on .NET Standard :)